resources/zib/zib-FreedomRestrictingIntervention.xml

```diff
@@ -0,0 +1,192 @@
+      <mapping>
+        <identity value="zib-freedomrestrictingintervention-v1.0-2020EN" />
+        <map value="NL-CM:14.6.8" />
+        <comment value="Start (implicit, main mapping is on `Procedure.performedPeriod`)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-freedomrestrictingintervention-v1.0-2020EN" />
+        <map value="NL-CM:14.6.9" />
+        <comment value="End (implicit, main mapping is on `Procedure.performedPeriod`)" />
+      </mapping>
+    </element>
+    <element id="Procedure.category">
+      <path value="Procedure.category" />
+      <min value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="225317005" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Procedure.code">
+      <path value="Procedure.code" />
+      <short value="TypeOfIntervention" />
+      <definition value="The type of intervention describes the kind of freedom-restricting intervention/involuntary care." />
+      <alias value="SoortInterventie" />
+      <min value="1" />
+      <binding>
+        <strength value="extensible" />
+        <description value="SoortInterventieCodelijsten" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.20--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-freedomrestrictingintervention-v1.0-2020EN" />
+        <map value="NL-CM:14.6.5" />
+        <comment value="TypeOfIntervention" />
+      </mapping>
+    </element>
+    <element id="Procedure.performed[x]">
+      <path value="Procedure.performed[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Procedure.performed[x]:performedPeriod">
+      <path value="Procedure.performed[x]" />
+      <sliceName value="performedPeriod" />
+      <min value="1" />
+      <type>
+        <code value="Period" />
+      </type>
+    </element>
+    <element id="Procedure.performed[x]:performedPeriod.start">
+      <path value="Procedure.performed[x].start" />
+      <short value="Start" />
+      <definition value="The date and time on which the intervention was carried out for point in time registrations or the date and time on which the intervention was started for duration registrations." />
+      <alias value="Begin" />
+      <min value="1" />
+      <mapping>
+        <identity value="zib-freedomrestrictingintervention-v1.0-2020EN" />

```

Comment:
Considering #52 - shall we place this not on a slice but on the `performed[x]` (if we agree that for this zib also a DateTime should be given if you don't intent a period.

Reply:
I added an extra slice on `.performedDateTime`, this seems like a more specific solution.

Reply:
Agreed. Need to updated https://github.com/Nictiz/Nictiz-R4-zib2020/issues/52 with this option. I did this for Procedure too.